### PR TITLE
PR: Help users understand why PySide 6.8.0 gives a segfault

### PR DESCRIPTION
**Pull request by hmaarrfk**
xref: https://github.com/spyder-ide/qtpy/issues/494

I'm not sure if you like to add things like this, but it is pretty hard to track this bug down.

### CODE REVIEW

File: qtpy/__init__.py [hunk 1]
```diff
@@ -278,6 +278,12 @@ def __init__(self, *, missing_package=None, **superclass_kwargs):
 if API in PYSIDE6_API:
     try:
         from PySide6 import __version__ as PYSIDE_VERSION  # analysis:ignore
+
+        if PYSIDE_VERSION == "6.8.0":
+            print(
+                "A known critical bug in PySide6 6.8.0 will cause your application to crash "
+                "see https://github.com/spyder-ide/qtpy/issues/494",
```
Comment (ccordoba12) [inline qtpy/__init__.py:285]:
```suggestion
                "A known critical bug in PySide6 6.8.0 will cause your application to crash. "
                "See https://github.com/spyder-ide/qtpy/issues/494",
```

File: qtpy/__init__.py [hunk 2]
```diff
@@ -278,6 +278,12 @@ def __init__(self, *, missing_package=None, **superclass_kwargs):
...
+            )
```
Comment (ccordoba12) [inline qtpy/__init__.py:286]: Shouldn't we add a `return` below this line to avoid the segfault?

**Review by ccordoba12 — COMMENTED** [review]
Thanks @hmaarrfk for your help with this!

File: qtpy/__init__.py (hunk 2 above)
Comment (ccordoba12) [reply qtpy/__init__.py:286]: Actually, that's not a good idea because it'll be generated in another module. So, ignore my comment.

Comment (hmaarrfk) [reply qtpy/__init__.py:286]: Yeah. I mean. I’m not trying to be fancy here. Just a bit of damage control.

Comment (hmaarrfk) [reply qtpy/__init__.py:286]: We shouldn’t overthink these things

Comment (ccordoba12) [reply qtpy/__init__.py:286]: Agreed. I'll release 2.4.2 with your fix during the weekend or on Monday at most.

**Review by ccordoba12 — APPROVED** [review]
Thanks @hmaarrfk!

**Pull request merged**